prometheus_hardware_exporter/collectors/ipmi_dcmi.py

```diff
@@ -11,6 +11,32 @@
 CURRENT_POWER_REGEX = re.compile(r"^Current Power\s*:\s*(?P<value>[0-9.]*)\s*Watts.*")
 
 
+class IpmiTool(Command):
+    """Command line tool for ipmitool."""
+
+    prefix = ""
+    command = "ipmitool"
+
+    def get_ps_redundancy(self) -> Tuple[bool, bool]:
+        """Get power supply redundancy.
+
+        returns:
+            - ok - Get redundancy success?
+            - redundancy - enable redundancy?
```

Comment:
This doesn't look very clear. Can you change the description to something like - 
`ok - True if fetching redundancy info is successful` 
`redundancy - True if redundancy is enabled`?